src/tools/rust-analyzer/crates/ide/src/syntax_highlighting/tests.rs

```diff
@@ -445,7 +445,7 @@ fn test_string_highlighting() {
 //- minicore: fmt, assert, asm, concat, panic
 macro_rules! println {
     ($($arg:tt)*) => ({
-        $crate::io::_print(format_args_nl!($($arg)*));
+        $crate::io::_print(std::format_args_nl!($($arg)*));
```

Comment:
This needs to be undone, the test does not run against the standard library here